src/cloud/components/WriteLimitOverlay.tsx

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useContext, FC} from 'react'
+import {useSelector} from 'react-redux'
+
+import {
+  OverlayContainer,
+  Overlay,
+  Form,
+  Heading,
+  Input,
+  HeadingElement,
+  Button,
+  ButtonType,
+  ButtonShape,
+  ComponentSize,
+  ComponentColor,
+  ComponentStatus,
+  InputType,
+} from '@influxdata/clockface'
+
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+// Selectors
+import {getOrg} from 'src/organizations/selectors'
+
+// Reporting
+import {event} from 'src/cloud/utils/reporting'
+
+// Design
+import './WriteLimitOverlay.scss'
+
+const WriteLimitOverlay: FC = () => {
+  const [limitReason, setLimitReason] = useState('')
+
+  const {onClose} = useContext(OverlayContext)
+
+  const {id} = useSelector(getOrg)
+
+  const handleSubmit = (): void => {
+    event('limit.requestincrease.writes', {reason: limitReason})
+
+    window.open(
+      `mailto:support@influxdata.com?subject=Request%20Query%20Write%20Limit%20Increase&body=Organization ID%3A%20${id}%0D%0A%0D%0AUse-Case Details:%0D%0A${limitReason}`
```

Comment:
just checking, the whitespace between Organization & ID are handled ok right? Same for Case Details

Reply:
Ignore this, based on the pics it looks like it's kosher